spec/controllers/assigned_badges_controller_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
```

Comment:
can we remove empty specs? it created illusion of having some tests for those controllers

Reply:
ah ok